code/modules/antagonists/blob/structures/_blob.dm

```diff
@@ -4,7 +4,7 @@
 	icon = 'icons/mob/blob.dmi'
 	light_range = 2
 	desc = "A thick wall of writhing tendrils."
-	density = FALSE //this being false causes two bugs, being able to attack blob tiles behind other blobs and being unable to move on blob tiles in no gravity, but turning it to 1 causes the blob mobs to be unable to path through blobs, which is probably worse.
```

Comment:
do we have JPS now? this is good to kill?

Reply:
I tested it, it worked just fine.